atc/atccmd/command.go

```diff
@@ -994,17 +999,36 @@ func (tripper mitmRoundTripper) RoundTrip(req *http.Request) (*http.Response, er
 	return tripper.RoundTripper.RoundTrip(req)
 }
 
-func (cmd *RunCommand) tlsConfig() (*tls.Config, error) {
+func (cmd *RunCommand) tlsConfig(logger lager.Logger, dbConn db.Conn) (*tls.Config, error) {
 	var tlsConfig *tls.Config
+	tlsConfig = atc.DefaultTLSConfig()
 
 	if cmd.isTLSEnabled() {
-		cert, err := tls.LoadX509KeyPair(string(cmd.TLSCert), string(cmd.TLSKey))
-		if err != nil {
-			return nil, err
-		}
+		tlsLogger := logger.Session("tls-enabled")
+		if cmd.EnableAutocert {
+			tlsLogger.Info("starting autocert manager")
```

Comment:
These log lines should match our [style guide](https://github.com/concourse/concourse/wiki/Concourse-Go-Style-Guide#logging) (and probably be at `Debug` level?):

```suggestion
			tlsLogger.Debug("using-autocert-manager")
```